requirements.txt

```diff
@@ -1,3 +1,4 @@
 numpy
 scipy
-xraydb
+xraydb<4.5.0
+sqlalchemy<2.0
```

Comment:
This looks like an unfortunate story

Reply:
https://github.com/pcdshub/pcdscalc/issues/21

Reply:
Matches up with what we're running on pcds-envs